packages/comms/src/services/wsSMC.ts

```diff
@@ -7,18 +7,161 @@ import { ESPConnection } from "../espConnection";
     * http://json2ts.com/
 */
 
-export namespace SMCActivity {
-    export interface Request {
-        ChatURL?: string;
-        BannerContent?: string;
-        BannerColor?: string;
-        BannerSize?: string;
-        BannerScroll?: string;
-        BannerAction?: string;
-        EnableChatURL?: boolean;
-        FromSubmitBtn?: boolean;
-        SortBy?: string;
-        Descending?: boolean;
+export namespace WsSMC {
+
+    export interface Activity {
+        ChatURL: string;
+        BannerContent: string;
+        BannerColor: string;
+        BannerSize: string;
+        BannerScroll: string;
+        BannerAction: number;
+        EnableChatURL: boolean;
+        FromSubmitBtn: boolean;
+        SortBy: string;
+        Descending: boolean;
+    }
+
+    export interface BrowseResources {
+    }
+
+    export interface ClearQueue {
+        Cluster: string;
+        QueueName: string;
+        Comment: string;
+        ServerType: string;
+        NetworkAddress: string;
+        Port: number;
+    }
+
+    export interface GetBuildInfo {
+    }
+
+    export interface GetStatusServerInfo {
+        ServerName: string;
+        ServerType: string;
+        NetworkAddress: string;
+        Port: number;
+    }
+
+    export interface GetThorQueueAvailability {
+    }
+
+    export interface Index {
+    }
+
+    export interface LockQuery {
+        EPIP: string;
+        XPath: string;
+        DurationMSLow: number;
+        DurationMSHigh: number;
+        TimeLockedLow: string;
+        TimeLockedHigh: string;
+        Mode: string;
+        AllFileLocks: boolean;
+    }
+
+    export interface MoveJobBack {
+        ClusterType: number;
+        Cluster: string;
+        QueueName: string;
+        Wuid: string;
+    }
+
+    export interface MoveJobDown {
+        ClusterType: number;
+        Cluster: string;
+        QueueName: string;
+        Wuid: string;
+    }
+
+    export interface MoveJobFront {
+        ClusterType: number;
+        Cluster: string;
+        QueueName: string;
+        Wuid: string;
+    }
+
+    export interface MoveJobUp {
+        ClusterType: number;
+        Cluster: string;
+        QueueName: string;
+        Wuid: string;
+    }
```

Comment:
Couldn't these 4 interfaces become one: `IMoveJob`?

Reply:
That is generated code from:  http://json2ts.com/